assets/src/theme/initializeThemeEditor.js

```diff
@@ -62,6 +62,7 @@ const setup = async () => {
     if (!event.altKey) {
       return;
     }
+    document.documentElement.classList.add('hide-wp-admin-bar');
```

Comment:
Don't we need to remove this at some point? 🤔

Reply:
Well it only happens after we enable the feature and after the user does the alt + mouseclick combination to open the theme editor. Probably at some point we can do this in a better way, but for now it seems better than having the admin bar make the site look different than it does for the end user.

Reply:
But is there a way for the user to close the theme editor?

Reply:
Yeah currently you can use the top left "hide" button to minimize it, but you can't really close it after opening. Though you can refresh the page. Not that closing and re-opening behavior are hard to add, but it didn't seem necessary at this stage so I just kept it simple.